pallets/circuit/src/tests.rs

```diff
@@ -3897,6 +3959,7 @@ fn execute_side_effects_with_xbi_works_for_call_evm() {
                 xtx_id,
                 valid_evm_sfx
                     .generate_id::<circuit_runtime_pallets::pallet_circuit::SystemHashing<Runtime>>(
```

Comment:
Could we have a test to cover the collision please?

Something like 2 Xtx's, different id, same indices on side effect, hash their ids, prove different.

Reply:
Done! (fingers crossed the pipeline passes on Friday evening right before I pack up)